alfresco-integration/shared/src/main/java/com/github/dynamicextensionsalfresco/models/DAOModelRegistrar.java

```diff
@@ -0,0 +1,52 @@
+package com.github.dynamicextensionsalfresco.models;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.LinkedList;
+import org.alfresco.repo.dictionary.DictionaryDAO;
+import org.alfresco.service.namespace.QName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Registers and unregisters [M2Model]s with a [DictionaryDAO]. This class is stateful and intended to
+ * support dynamic loading of models.
+ *
+ * @author Laurens Fridael
+ * @author Toon Geens
+ */
+
+public class DAOModelRegistrar extends AbstractModelRegistrar {
+
+    private Logger logger = LoggerFactory.getLogger(DAOModelRegistrar.class);
```

Comment:
can you make your logger `private static final`.

Just to keep up with the "Java conventions"

Reply:
@stanmine This was intentional

1. This is a straight port from Kotlin `private val logger` - which does not imply a `static` modifier (yes, it should be `final`)
2. There are potentially even issues making this `static` when doing funny things with classloaders - see https://wiki.apache.org/commons/Logging/StaticLog for example - at this point I'm not sure this applies to DE or not

Reply:
This is indeed a concern if you would deploy 2 alfresco's with DE on the same application container. And with all DE jars in the shared class loader. If the logger would be static in this case you will only be able to set the logging level for both alfresco's simultaneous.

Making the logger static is "conventional" due to the memory and cpu overhead gain it brings. However, in modern environments this will be less of an issue.

The important thing here, is just to make sure that it is consistency across all the DE code.
a quick lookup learns that all types of logging declarations are used (`private final static Logger logger`, `private Logger logger`,  `private final Logger logger`). So perhaps we should stick to `private final Logger logger` for all.

Marked as fixed :-)